impeller/renderer/backend/metal/pipeline_library_mtl.mm

```diff
@@ -112,7 +112,8 @@
       ^(id<MTLRenderPipelineState> _Nullable render_pipeline_state,
         NSError* _Nullable error) {
         if (error != nil) {
-          VALIDATION_LOG << "Could not create render pipeline: "
```

Comment:
This change would have made the error a lot easier to find for someone that isn't building the engine.

Reply:
This is a nice add. 👍